src/base-apis.js

```diff
@@ -1888,6 +1902,25 @@ MatrixBaseApis.prototype.getThirdpartyUser = function(protocol, params) {
     );
 };
 
+MatrixBaseApis.prototype.getTerms = function(serviceType, baseUrl, accessToken) {
+    const url = termsUrlForService(serviceType, baseUrl);
+    return this._http.requestOtherUrl(
+        undefined, 'GET', url, null, null, null,
+    );
+};
+
+MatrixBaseApis.prototype.agreeToTerms = function(
+    serviceType, baseUrl, accessToken, termsUrls,
+) {
+    const url = termsUrlForService(serviceType, baseUrl);
+    const headers = {
+        Authorization: "Bearer " + accessToken,
+    };
+    return this._http.requestOtherUrl(
+        undefined, 'POST', url, null, {user_accepts: termsUrls}, { headers },
```

Comment:
Nit: Please choose one style of spacing in an object, at least so this line is internally consistent 😜